Sprint-2/debug/2.js

```diff
@@ -1,8 +1,10 @@
 // Predict and explain first...
+//returns 3 for every number 
+//we should pass num variable as a parameter in function rather than assigning a value to num outside of the function
```

Comment:
Yes - what do we call this scope outside the function?

Reply:
The scope outside of any function is called global scope.